crates/wasi-common/src/sys/osother.rs

```diff
@@ -0,0 +1,90 @@
+use super::sys_impl::oshandle::OsHandle;
+use super::{fd, AsFile};
+use crate::handle::{Handle, HandleRights};
+use crate::sandboxed_tty_writer::SandboxedTTYWriter;
+use crate::wasi::types::{self, Filetype};
+use crate::wasi::Result;
+use std::any::Any;
+use std::cell::Cell;
+use std::fs::File;
+use std::io::{self, Read, Write};
+use std::ops::Deref;
+
+pub(crate) trait OsOtherExt {
+    /// Create `OsOther` as `dyn Handle` from null device.
+    fn from_null() -> io::Result<Box<dyn Handle>>;
+}
+
+#[derive(Debug)]
```

Comment:
Can you add a comment here briefly explaining what `OsOther` is?

Reply:
Oh right, sure thing!

Reply:
Done!